cypress/integration/tests/signIn/signIn.js

```diff
@@ -2,11 +2,57 @@ import SignIn from '../../../page-objects/signInPage';
 import credentials from '../../../fixtures/credentials.json';
 
 describe('Sign In', () => {
-  it('TA-1: User is able to log in with valid credentials', () => {
-    SignIn.navigateToSignInPage();
-    SignIn.enterSignInEmail(credentials.email);
-    SignIn.enterSignInPassword(credentials.password);
+
+  it('TA-1: User is able to sign in with valid credentials', () => {
+    SignIn.open('/signing');
```

Comment:
you are doing `SignIn.open('/signing');` before each test, so please reorganize with https://docs.cypress.io/guides/core-concepts/writing-and-organizing-tests.html#Hooks

Reply:
done